spec/parsers/maven_spec.rb

```diff
@@ -422,19 +422,20 @@
 
       runtime_classpath = deps[:dependencies].select {|item| item[:type] == "runtimeClasspath"}
 
-      expect(runtime_classpath.length).to eq 156
+      expect(runtime_classpath.length).to eq 157
       expect(runtime_classpath.select {|item| item[:name] == "com.google.guava:guava"}.length).to eq 1
 
-      # test rename resolution
-      renamed_dep = runtime_classpath.
-        select do |item| 
-          item[:name] == "commons-io:commons-io" && 
-          item[:original_name] == "apache:commons-io" &&
-          item[:requirement] == "2.6" &&
-          item[:original_requirement] == "1.4" 
+      # test rename resolutions
+      [
+        {name: "commons-io:commons-io", requirement: "2.6", original_name: "apache:commons-io", original_requirement: "1.4"},
+        {name: "axis:axis", requirement: "1.4", original_name: "apache:axis", original_requirement: "*"}
```

Comment:
We also default a nil requirement in `build.gradle` files to `*` so I'm using `*` here too, instead of nil.